prime-router/src/main/kotlin/serializers/Hl7Serializer.kt

```diff
@@ -397,12 +397,16 @@ class Hl7Serializer(
         val mapping = convertBatchMessagesToMap(messageBody, schema, sender = sender)
         val mappedRows = mapping.mappedRows
 
+        // Generate the action log
+        val actionLogs = ActionLogger()
+
+        // throw an error if the max items limit is exceeded
+        SerializerValidation.throwMaxItemsError(mappedRows.size, actionLogs)
```

Comment:
i am not sure we want to keep this. this is an example of the shared max size validator.
we can make this even more configurable by making the validator take a max size parameter. 
that way each Serializer can have its own max size without affecting the other.
right now, we're simply using the defaults that were in place before